fix(hds compliance): add a FAQ entry about 'point de contact' (#1943)
Scalingo/documentation: src/_posts/compliance/2000-01-01-hds.md

[Download the current HDS 1.1 certificate](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)

> # 🇬🇧 What is a "Point of Contact health Professional" ("Point de contact professionnel de santé" in French)? {#en-hds-point-of-contact-health-professional}

Reviewer: ```suggestion
# 🇬🇧 What is a "Health Professional Point of Contact" ("Point de contact professionnel de santé" in French)? {#en-hds-point-of-contact-health-professional}
```

> The HDS certification enjoins Scalingo to maintain a list of *Points of contact health professional* for each of its customer.

Reviewer 2: ```suggestion
The HDS certification enjoins Scalingo to have a up-to-date *Health Professional Point of Contact* for each of its customers.
```

Author: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

> These points of contact must be able to designate an health professional to Scalingo when required (for example, to access to health data or to manage the relationship with the client, ...)

Reviewer 2: ```suggestion
This Point of Contact is a person able to designate a Health Professional to Scalingo when the situation requires it (for example, to access to Health Data or to manage a Security Incident, ...)
```

Author: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

> This Point of Contact is a person able to designate a Health Professional to Scalingo when the situation requires it (for example, to access to Health Data or to manage a Security Incident, ...)

Reviewer: ```suggestion
This Point of Contact is a person able to designate a Health Professional to Scalingo when the situation requires it (for example, to access Health Data or to manage a Security Incident, ...)
```

La certification HDS de Scalingo couvre les périmètres suivants :

> 1. Mise à disposition et maintien en condition opératThe search found this: ionnelle des sites physiques permettant d’héberger l’infrastructure matérielle du système d’information utilisé pour le traitement des données de santé ;

Reviewer 2: ;)

> 1. Mise à disposition et maintien en condition opératThe search found this: ionnelle des sites physiques permettant d’héberger l’infrastructure matérielle du système d’information utilisé pour le traitement des données de santé ;

Author: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

Sorry for this :(

Veuillez toujours vous référer à la page actuelle pour obtenir les dernières informations sur notre conformité HDS

[Télécharger le certificat HDS 1.1 en vigueur](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)

## 🇫🇷 Qu'est-ce qu'un "Point de contact professionnel de santé" ? {#fr-hds-point-of-contact-health-professional}

La certification HDS impose à Scalingo de maintenir une liste de *points de contact professionnel de santé* pour chacun de ses clients.

> Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer des relations avec le patient, etc.)

Reviewer 2: ```suggestion
Le rôle de ces points de contact est d'être en mesure de désigner à Scalingo un professionnel de santé lorsque cela est nécessaire (par exemple, pour accéder aux données de santé, pour gérer un incident de sécurité, etc.)
```

Author: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/3943b211ac12226b8a28cb9e5049bf091371b1f2

> En outre, Scalingo doit être en capacité de transmettre sans délai cette liste à l’autorité compétente sur demande, notamment en cas de suspension ou de retrait de la certification HDS.

Reviewer 2: Source ?

Author: Page 17 of this pdf file, which seems legit: https://esante.gouv.fr/sites/default/files/media_entity/documents/hds_referentiel_de_certification_v1.1f_mai2018.pdf

Please tell me if I should use another source :)

Reviewer 2: You couldn't find better source in fact :)
I suggest you mention it in the french and english versions (there is an english translation of the referential too)

Reference it like this

> En outre, Scalingo doit être en capacité de transmettre sans délai cette liste à l’autorité compétente sur demande, notamment en cas de suspension ou de retrait de la certification HDS (Référentiel HDS 1.1.1f, exigence 4.5.4).

Author: Should I add a link the referential? (if so, could you please provide them?)

Reviewer: @Frzk IMHO linking the referential is not necessary. But you should mention the referential article number like Yannick did ie "Référentiel HDS 1.1.1f, exigence 4.5.4"

Author: Fixed in https://github.com/Scalingo/documentation/pull/1943/commits/d5252491897da1d156b7e0e5ea7e81b8e7027781
